### ISSUE TITLE
[Feature request] Profile Manager

### ISSUE THREAD

**Issue by mrnapz** [jlesage/docker-firefox#215]
### Idea

Hi would be possible to add the profile manager in order to create multiple profiles on the same container? 

Thank you in advance.

**Comment by IgitBuh** [comment]
Isn't the built-in profile manager doing that already?
`about:profiles`

You can create and open a different profile there.

I haven't tried passing launch parameters to the container, but `-P "profilename"` would run Firefox with the specified profile.

**Comment by imnetcat** [comment]
+1 for this and I will add some more info bellow:
1. Container launching firefox with this command: `/usr/lib/firefox/firefox --profile /config/profile --setDefaultBrowser`
1) this command generated there in params file: https://github.com/jlesage/docker-firefox/blob/master/rootfs/etc/services.d/app/params
2) I cannot see flag `--profile` in current FF command line documentation: https://wiki.mozilla.org/Firefox/CommandLineOptions
There is `-profile` flag or `-P` tho

2. I tried to pass env var `FF_CUSTOM_ARGS="-no-remote -P profile21"` (or just `FF_CUSTOM_ARGS="-P profile21"` tried too) and launch the container. As you can see arguments are passed to FF and FF launched:
![Image](https://github.com/user-attachments/assets/97aa2fef-0f20-42cf-9328-0da013a7d39e)

3. But when I open the launched FF instance: profile21 is not opened nor an default or default-release one, like we launched the incognito-profile:

![Image](https://github.com/user-attachments/assets/0f811e6b-1952-44eb-8b8e-361d1e8bf931)

4. In case I click the button in FF UI then FF relaunched with this profile as expected:

![Image](https://github.com/user-attachments/assets/f77e292d-29c8-4d2e-ad09-204e045bc454)

![Image](https://github.com/user-attachments/assets/c63ac477-b814-4bf9-b93e-936682a6acaa)

5. and I will point out that the `--profile /config/profile` arg passed to FF by default docker image confusing me so much because actual profile folder FF find out in another place: 

![Image](https://github.com/user-attachments/assets/07e18360-d8f5-494d-903c-6396b39df681)

**In conclusion**: I can't launch FF with a specific profile. This can be an issue or a feature to do
